packages/protobuf-test/src/mixing-instances.test.ts

```diff
@@ -37,14 +35,12 @@ describe("mixing message instances", () => {
 });
 
 describe("mixing message instances in the constructor", () => {
-  test("normalizes by creating a new instance", () => {
+  test("matches expected message", () => {
     const test = new JS_MessageFieldMessage_TestMessage({ name: "foo" });
     const message = new TS_MessageFieldMessage({
       messageField: test,
     });
     expect(message.messageField?.name).toBe("foo");
-    expect(message.messageField).toBeInstanceOf(
-      TS_MessageFieldMessage_TestMessage,
-    );
+    expect(isMessage(message.messageField, JS_MessageFieldMessage_TestMessage));
```

Comment:
We test what happens when a message field is passed to a constructor. Is it kept as is if it's the same type?

Our understanding of "same type" changed slightly. To test more precisely, let's not use `isMessage`, but an identity check instead: 

```suggestion
    expect(message.messageField).toBe(test);
```